### DISCUSSION TITLE
It can only see part of my database, how do I fix it?

### DISCUSSION THREAD

**Discussion by lilskippyy** [FlowiseAI/Flowise#3666] [Q&A]
Hello. Actually, that happens to other rag apps i tried , so not a Flowise issue I think.

For example, when I create a database with a csv file with a few hundred entries, when asking the LLM for any type analysis it always says it can only see 3 or 4 entries.

It's probably some setting I don't know, or maybe its my system that is not good enough?

I have 8gb of vram, and using ollama with llama3.2-vison.

Any tips to help me solve that are welcome . Thank you all so much!

**Comment by bernejorge** [comment]
When you define the vector store there are an property called Top K. In this property you define the number of registries will be retrived. By default Top k = 4.
![image](https://github.com/user-attachments/assets/d51c9168-a043-4ade-ac6d-150286adfb44)

**Reply by lilskippyy** [reply]
That worked thanks.
Now , any tips to make it come back with more accurate answers?

For example, when I ask what are the top 10 highest prices on, a list it always come up with some random items instead of the top priced ones.
